airbyte-integrations/connectors/source-mixpanel/source_mixpanel/spec.json

```diff
@@ -4,9 +4,19 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Source Mixpanel Spec",
     "type": "object",
-    "required": ["api_secret"],
     "additionalProperties": true,
     "properties": {
+      "serviceaccount_username": {
+        "title": "Mixpanel Service Account username",
+        "type": "string",
+        "description": "Mixpanel Service Account username"
```

Comment:
Please look other connectors implementation how they use oneOf configuration to be able to select between one type of another. Check Github spec.json file or search for "credentials" in spec.json files in your editor